src/savegame_json.cpp

```diff
@@ -718,6 +721,7 @@ void Character::store( JsonOut &json ) const
 
     json.member( "base_age", init_age );
     json.member( "base_height", init_height );
+    json.member_as_string( "blood_type", my_blood_type );
```

Comment:
It looks like you aren't serializing / deserializing the `blood_rh_factor`.

Reply:
Indeed, thanks for spotting it. I've split the blood type and Rh factor and apparently forgot didn't notice the save/load doesn't cover both. Will fix ASAP.

Reply:
Fix in #41016